sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectName=RPE :: expressjs template
+sonar.projectKey=RPEET
```

Comment:
seems like a weird key?

Reply:
I follow the same naming convention that in I&A.

Reply:
I think this would be better?
```suggestion
sonar.projectKey=rpe-expressjs-template
```